src/pages/user/[userid].jsx

```diff
@@ -0,0 +1,34 @@
+import { useRouter } from 'next/router'
+import { useState } from 'react'
+import { Formik } from 'formik'
+
+const User = () => {
+  // const router = useRouter()
+  // const { userid } = router.query
+  const [name, setName] = useState('')
+  const [email, setEmail] = useState('')
+  const [phone, setPhone] = useState('')
+  const [resume, setResume] = useState('')
+
+  const handleSubmit = () => {
+    console.log(name, email, phone, resume)
+  }
+
+  return (
+    <div>
+      <div>
+        <h1>Input your name</h1>
```

Comment:
Using too many `<h1>` tags generally results in poorer SEO.

Perhaps we can use a label here if there isn't one already?

Reply:
Forgot to delete the user page, it was just a test. Haven't started implementing it.